io-onboarding-pa-api/templates/deployment.yaml

```diff
@@ -74,6 +74,18 @@ spec:
               value: "{{ .Values.ioOnboardingPaApi.env.saml_cert_path }}"
             - name: TOKEN_DURATION_IN_SECONDS
               value: "{{ .Values.ioOnboardingPaApi.env.token_duration_in_seconds }}"
+            - name: EMAIL_PASSWORD
```

Comment:
Instead of this -after communicating the new password to me- you should add:

-name: EMAIL_PASSWORD
  value: k8s-io-onboarding-pa-api-secrets@azurekeyvault?email-password